Assets/HoloToolkit/Build/Editor/HoloToolkitCommands.cs

```diff
@@ -1,4 +1,9 @@
-﻿using HoloToolkit.Unity;
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think this is definitely worth a new issue for tracking, maybe get LCA or the lead to track this down?